pkg/operation/hybridbotanist/cloud_config.go

```diff
@@ -78,10 +78,11 @@ func (b *HybridBotanist) generateCloudConfigChart() (*chartrenderer.RenderedChar
 			},
 			"version": b.Shoot.Info.Spec.Kubernetes.Version,
 		},
-		"workers": workers,
+		"workers":    workers,
+		"kubeconfig": b.Secrets["cloud-config-downloader"].Data["kubeconfig"],
```

Comment:
Why is this here? Seems not to be used anywhere, nor is it part of the `values.yaml`.